src/app/actions/requestOverflowAction/index.ts

```
@@ -12,7 +13,7 @@ import type { TaskActionValue } from "@/app/SlackBlocks";
  * @returns {GoogleAppsScript.Content.TextOutput} レスポンス
  */
 export function requestOverflowAction(payload: any): GoogleAppsScript.Content.TextOutput {
-  logDebug("requestOverflowAction");
+  // logDebug("requestOverflowAction");
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)